notebooks/pipeline.config.yaml

```diff
@@ -0,0 +1,14 @@
+pipelines:
+- name: rtcs
+  compile: qat.pipelines.DefaultCompile
+  execute: qat.pipelines.DefaultExecute
+  hardware:
+    hardware_type: rtcs
+  default: false
+- name: echo
+  compile: qat.pipelines.DefaultCompile
+  execute: qat.pipelines.DefaultExecute
+  hardware:
+    qubit_count: 8
+    hardware_type: echo
+  default: true
```

Comment:
mere observation: pipelines seem to be named after the hw/engine they're gonna target. Could there be other conceptual characteristics to specifying a pipeline ? 🤔

Reply:
Certainly, this is just an example :)